Libraries/LibX86/Instruction.cpp

```diff
@@ -199,6 +200,28 @@ static void build_slash(InstructionDescriptor* table, u8 op, u8 slash, const cha
     build(d.slashes, slash, mnemonic, format, handler, lock_prefix_allowed);
 }
 
+static void build_slash_rm(u8 op, u8 slash, u8 rm, const char* mnemonic, InstructionFormat format, InstructionHandler handler, IsLockPrefixAllowed lock_prefix_allowed = LockPrefixNotAllowed)
+{
+    ASSERT((rm & 0xc0) == 0xc0);
+    ASSERT(((rm >> 3) & 7) == slash);
+    InstructionDescriptor* table = s_table32;  // FIXME: Is there a 16-bit chip with FPU instrs?
```

Comment:
`s_table16` vs `s_table32` is not about the chip type, but rather which operand size is currently in effect.
`mov ax, bx` and `mov eax, ebx` are the same opcode, just with different operand sizes.
UserspaceEmulator currently runs with 32-bit operands by default, but an operand size override prefix (`0x66`) turns it into a 16-bit op and then we look in `s_table16` instead. :)

Reply:
Is there a way I can test this (other than writing a machine code file by hand, I suppose)? I tried `BITS 16` in nasm, but that prefixes x87 instructions that point to memory with 0x67, not 0x66, and it confuses objdump, so maybe nasm doesn't do it right.

Reply:
You can always force out a prefix by just doing `db 0x66`

Reply:
Ah, right, thanks. Well, it seems to work :)